[2023-11-16T10:30:48.532+0000] {processor.py:157} INFO - Started process (PID=180) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:30:48.533+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:30:48.534+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:30:48.534+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:30:49.377+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:30:49.386+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.856 seconds
[2023-11-16T10:31:19.552+0000] {processor.py:157} INFO - Started process (PID=189) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:19.552+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:31:19.554+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:31:19.553+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:19.889+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:19.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.348 seconds
[2023-11-16T10:31:50.047+0000] {processor.py:157} INFO - Started process (PID=198) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:50.047+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:31:50.049+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:31:50.049+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:50.405+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:31:50.414+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.370 seconds
[2023-11-16T10:32:20.566+0000] {processor.py:157} INFO - Started process (PID=207) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:20.567+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:32:20.568+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:32:20.568+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:20.909+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:20.917+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.353 seconds
[2023-11-16T10:32:51.074+0000] {processor.py:157} INFO - Started process (PID=216) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:51.074+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:32:51.076+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:32:51.076+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:51.408+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:32:51.416+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.344 seconds
[2023-11-16T10:33:21.571+0000] {processor.py:157} INFO - Started process (PID=225) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:21.573+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:33:21.574+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:33:21.574+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:21.892+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:21.901+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.331 seconds
[2023-11-16T10:33:52.057+0000] {processor.py:157} INFO - Started process (PID=234) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:52.058+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:33:52.060+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:33:52.060+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:52.399+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:33:52.407+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.352 seconds
[2023-11-16T10:34:22.822+0000] {processor.py:157} INFO - Started process (PID=243) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:22.823+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:34:22.825+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:34:22.825+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:23.173+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:23.180+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.360 seconds
[2023-11-16T10:34:53.335+0000] {processor.py:157} INFO - Started process (PID=252) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:53.336+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:34:53.344+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:34:53.344+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:53.673+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:34:53.681+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.347 seconds
[2023-11-16T10:35:23.840+0000] {processor.py:157} INFO - Started process (PID=261) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:23.841+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:35:23.843+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:35:23.843+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:24.176+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:24.184+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.346 seconds
[2023-11-16T10:35:54.338+0000] {processor.py:157} INFO - Started process (PID=270) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:54.338+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:35:54.340+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:35:54.339+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:54.685+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:35:54.693+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.357 seconds
[2023-11-16T10:36:24.851+0000] {processor.py:157} INFO - Started process (PID=279) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:24.852+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:36:24.853+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:36:24.853+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:25.181+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:25.189+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.341 seconds
[2023-11-16T10:36:55.352+0000] {processor.py:157} INFO - Started process (PID=288) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:55.353+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:36:55.354+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:36:55.354+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:55.687+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:36:55.696+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.347 seconds
[2023-11-16T10:37:25.844+0000] {processor.py:157} INFO - Started process (PID=297) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:25.845+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:37:25.847+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:37:25.846+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:26.209+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:26.218+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.376 seconds
[2023-11-16T10:37:56.368+0000] {processor.py:157} INFO - Started process (PID=306) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:56.369+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:37:56.370+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:37:56.370+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:56.729+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:37:56.737+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.372 seconds
[2023-11-16T10:38:26.893+0000] {processor.py:157} INFO - Started process (PID=315) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:26.894+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:38:26.896+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:38:26.895+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:27.235+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:27.244+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.353 seconds
[2023-11-16T10:38:57.398+0000] {processor.py:157} INFO - Started process (PID=324) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:57.399+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:38:57.401+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:38:57.400+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:57.747+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:38:57.755+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.359 seconds
[2023-11-16T10:39:27.909+0000] {processor.py:157} INFO - Started process (PID=333) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:27.909+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:39:27.911+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:39:27.911+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:28.238+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:28.246+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.340 seconds
[2023-11-16T10:39:58.410+0000] {processor.py:157} INFO - Started process (PID=342) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:58.411+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:39:58.412+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:39:58.412+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:58.754+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:39:58.762+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.355 seconds
[2023-11-16T10:40:28.919+0000] {processor.py:157} INFO - Started process (PID=351) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:28.920+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:40:28.922+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:40:28.922+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:29.270+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:29.280+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.363 seconds
[2023-11-16T10:40:59.434+0000] {processor.py:157} INFO - Started process (PID=360) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:59.434+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:40:59.436+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:40:59.436+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:59.761+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:40:59.769+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.338 seconds
[2023-11-16T10:41:29.921+0000] {processor.py:157} INFO - Started process (PID=369) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:41:29.922+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:41:29.924+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:41:29.923+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:41:30.277+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:41:30.285+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.366 seconds
[2023-11-16T10:42:00.435+0000] {processor.py:157} INFO - Started process (PID=378) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:00.436+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:42:00.438+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:42:00.437+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:00.761+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:00.769+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.336 seconds
[2023-11-16T10:42:30.940+0000] {processor.py:157} INFO - Started process (PID=387) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:30.941+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:42:30.943+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:42:30.942+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:31.276+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:42:31.285+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.347 seconds
[2023-11-16T10:43:01.441+0000] {processor.py:157} INFO - Started process (PID=396) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:01.442+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:43:01.444+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:43:01.443+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:01.783+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:01.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.353 seconds
[2023-11-16T10:43:31.940+0000] {processor.py:157} INFO - Started process (PID=405) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:31.941+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:43:31.942+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:43:31.942+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:32.268+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:43:32.276+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.338 seconds
[2023-11-16T10:44:02.423+0000] {processor.py:157} INFO - Started process (PID=414) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:02.423+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:44:02.425+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:44:02.425+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:02.769+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:02.777+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.357 seconds
[2023-11-16T10:44:32.930+0000] {processor.py:157} INFO - Started process (PID=423) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:32.930+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:44:32.932+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:44:32.932+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:33.244+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:44:33.252+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.324 seconds
[2023-11-16T10:45:03.401+0000] {processor.py:157} INFO - Started process (PID=432) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:03.402+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:45:03.404+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:45:03.404+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:03.765+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:03.774+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.375 seconds
[2023-11-16T10:45:33.933+0000] {processor.py:157} INFO - Started process (PID=441) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:33.934+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:45:33.936+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:45:33.935+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:34.269+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:45:34.277+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.346 seconds
[2023-11-16T10:46:04.449+0000] {processor.py:157} INFO - Started process (PID=450) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:04.450+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:46:04.451+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:46:04.451+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:04.780+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:04.789+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.342 seconds
[2023-11-16T10:46:34.939+0000] {processor.py:157} INFO - Started process (PID=459) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:34.940+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:46:34.942+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:46:34.942+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:35.281+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:46:35.289+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.352 seconds
[2023-11-16T10:47:05.436+0000] {processor.py:157} INFO - Started process (PID=468) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:05.437+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:47:05.439+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:47:05.438+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:05.759+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:05.767+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.333 seconds
[2023-11-16T10:47:35.921+0000] {processor.py:157} INFO - Started process (PID=477) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:35.922+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:47:35.924+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:47:35.924+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:36.259+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:47:36.267+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.348 seconds
[2023-11-16T10:48:06.413+0000] {processor.py:157} INFO - Started process (PID=486) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:06.414+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:48:06.415+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:48:06.415+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:06.751+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:06.760+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.349 seconds
[2023-11-16T10:48:36.910+0000] {processor.py:157} INFO - Started process (PID=495) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:36.911+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:48:36.912+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:48:36.912+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:37.229+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:48:37.237+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.329 seconds
[2023-11-16T10:49:07.404+0000] {processor.py:157} INFO - Started process (PID=504) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:07.405+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:49:07.407+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:49:07.407+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:07.737+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:07.745+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.343 seconds
[2023-11-16T10:49:37.909+0000] {processor.py:157} INFO - Started process (PID=513) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:37.910+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:49:37.912+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:49:37.911+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:38.226+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:49:38.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.328 seconds
[2023-11-16T10:50:08.398+0000] {processor.py:157} INFO - Started process (PID=522) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:08.399+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:50:08.400+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:50:08.400+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:08.718+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:08.726+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.331 seconds
[2023-11-16T10:50:38.884+0000] {processor.py:157} INFO - Started process (PID=531) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:38.885+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:50:38.887+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:50:38.886+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:39.203+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:50:39.211+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.329 seconds
[2023-11-16T10:51:09.378+0000] {processor.py:157} INFO - Started process (PID=547) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:09.378+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:51:09.380+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:51:09.380+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:09.718+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:09.725+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.350 seconds
[2023-11-16T10:51:39.876+0000] {processor.py:157} INFO - Started process (PID=556) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:39.877+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:51:39.879+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:51:39.879+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:40.204+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:51:40.212+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.338 seconds
[2023-11-16T10:52:10.368+0000] {processor.py:157} INFO - Started process (PID=565) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:10.368+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:52:10.370+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:52:10.370+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:10.686+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:10.694+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.328 seconds
[2023-11-16T10:52:40.853+0000] {processor.py:157} INFO - Started process (PID=574) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:40.853+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:52:40.855+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:52:40.855+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:41.236+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:52:41.245+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.395 seconds
[2023-11-16T10:53:11.710+0000] {processor.py:157} INFO - Started process (PID=583) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:11.712+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:53:11.717+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:53:11.716+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:13.055+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:13.071+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.364 seconds
[2023-11-16T10:53:43.335+0000] {processor.py:157} INFO - Started process (PID=592) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:43.336+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:53:43.338+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:53:43.338+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:43.853+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:53:43.868+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.536 seconds
[2023-11-16T10:54:14.143+0000] {processor.py:157} INFO - Started process (PID=601) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:14.144+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:54:14.146+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:54:14.146+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:14.777+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:14.795+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.655 seconds
[2023-11-16T10:54:45.046+0000] {processor.py:157} INFO - Started process (PID=610) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:45.048+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:54:45.051+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:54:45.050+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:45.690+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:54:45.704+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.661 seconds
[2023-11-16T10:55:15.968+0000] {processor.py:157} INFO - Started process (PID=619) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:15.969+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:55:15.972+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:55:15.972+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:16.587+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:16.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.639 seconds
[2023-11-16T10:55:46.851+0000] {processor.py:157} INFO - Started process (PID=628) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:46.852+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:55:46.855+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:55:46.854+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:47.502+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:55:47.516+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.668 seconds
[2023-11-16T10:56:17.752+0000] {processor.py:157} INFO - Started process (PID=637) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:17.753+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:56:17.757+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:56:17.756+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:18.479+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:18.496+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.749 seconds
[2023-11-16T10:56:48.782+0000] {processor.py:157} INFO - Started process (PID=646) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:48.783+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:56:48.787+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:56:48.786+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:49.434+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:56:49.448+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.670 seconds
[2023-11-16T10:57:19.868+0000] {processor.py:157} INFO - Started process (PID=655) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:19.869+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:57:19.872+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:57:19.872+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:20.887+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:20.904+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.039 seconds
[2023-11-16T10:57:51.222+0000] {processor.py:157} INFO - Started process (PID=664) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:51.224+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:57:51.226+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:57:51.226+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:51.795+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:57:51.808+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.591 seconds
[2023-11-16T10:58:22.076+0000] {processor.py:157} INFO - Started process (PID=673) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:22.077+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:58:22.079+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:58:22.078+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:23.934+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:23.949+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.876 seconds
[2023-11-16T10:58:54.223+0000] {processor.py:157} INFO - Started process (PID=682) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:54.225+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:58:54.228+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:58:54.228+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:54.928+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:58:54.946+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.727 seconds
[2023-11-16T10:59:25.201+0000] {processor.py:157} INFO - Started process (PID=691) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:25.203+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:59:25.207+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:59:25.206+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:26.023+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:26.037+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.840 seconds
[2023-11-16T10:59:56.151+0000] {processor.py:157} INFO - Started process (PID=700) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:56.153+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T10:59:56.156+0000] {logging_mixin.py:151} INFO - [2023-11-16T10:59:56.156+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:56.897+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T10:59:56.913+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.767 seconds
[2023-11-16T11:00:27.200+0000] {processor.py:157} INFO - Started process (PID=709) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:27.202+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:00:27.205+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:00:27.205+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:27.926+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:27.942+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.747 seconds
[2023-11-16T11:00:58.572+0000] {processor.py:157} INFO - Started process (PID=718) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:58.574+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:00:58.577+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:00:58.577+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:59.248+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:00:59.262+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.694 seconds
[2023-11-16T11:01:29.557+0000] {processor.py:157} INFO - Started process (PID=727) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:01:29.559+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:01:29.562+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:01:29.562+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:01:30.128+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:01:30.142+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.590 seconds
[2023-11-16T11:02:00.242+0000] {processor.py:157} INFO - Started process (PID=736) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:00.244+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:02:00.248+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:02:00.247+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:01.012+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:01.029+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.791 seconds
[2023-11-16T11:02:31.301+0000] {processor.py:157} INFO - Started process (PID=745) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:31.302+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:02:31.305+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:02:31.304+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:32.015+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:02:32.036+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.739 seconds
[2023-11-16T11:03:02.295+0000] {processor.py:157} INFO - Started process (PID=754) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:02.303+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:03:02.306+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:03:02.305+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:02.970+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:02.983+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.692 seconds
[2023-11-16T11:03:33.047+0000] {processor.py:157} INFO - Started process (PID=763) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:33.048+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:03:33.050+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:03:33.050+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:33.589+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:03:33.602+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.559 seconds
[2023-11-16T11:04:03.860+0000] {processor.py:157} INFO - Started process (PID=773) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:03.863+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:04:03.867+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:04:03.866+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:04.607+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:04.625+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.769 seconds
[2023-11-16T11:04:34.871+0000] {processor.py:157} INFO - Started process (PID=782) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:34.872+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:04:34.875+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:04:34.874+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:35.466+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:04:35.481+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.614 seconds
[2023-11-16T11:05:05.751+0000] {processor.py:157} INFO - Started process (PID=791) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:05.753+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:05:05.756+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:05:05.756+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:06.372+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:06.386+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.640 seconds
[2023-11-16T11:05:36.629+0000] {processor.py:157} INFO - Started process (PID=800) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:36.630+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:05:36.633+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:05:36.633+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:37.231+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:05:37.246+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.620 seconds
[2023-11-16T11:06:07.569+0000] {processor.py:157} INFO - Started process (PID=809) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:07.571+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:06:07.576+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:06:07.575+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:08.189+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:08.201+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.637 seconds
[2023-11-16T11:06:38.460+0000] {processor.py:157} INFO - Started process (PID=818) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:38.461+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:06:38.463+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:06:38.463+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:39.050+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:06:39.063+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.607 seconds
[2023-11-16T11:07:09.361+0000] {processor.py:157} INFO - Started process (PID=827) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:09.363+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:07:09.369+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:07:09.369+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:10.080+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:10.095+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.739 seconds
[2023-11-16T11:07:40.368+0000] {processor.py:157} INFO - Started process (PID=836) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:40.369+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:07:40.373+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:07:40.372+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:40.967+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:07:40.982+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.618 seconds
[2023-11-16T11:08:11.137+0000] {processor.py:157} INFO - Started process (PID=852) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:11.138+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:08:11.142+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:08:11.141+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:11.770+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:11.786+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.653 seconds
[2023-11-16T11:08:42.060+0000] {processor.py:157} INFO - Started process (PID=861) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:42.061+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:08:42.064+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:08:42.064+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:42.706+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:08:42.722+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.666 seconds
[2023-11-16T11:09:12.981+0000] {processor.py:157} INFO - Started process (PID=870) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:12.982+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:09:12.985+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:09:12.985+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:13.598+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:13.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.637 seconds
[2023-11-16T11:09:43.889+0000] {processor.py:157} INFO - Started process (PID=879) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:43.891+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:09:43.893+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:09:43.893+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:44.561+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:09:44.579+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.693 seconds
[2023-11-16T11:10:12.431+0000] {processor.py:157} INFO - Started process (PID=888) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:12.432+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:10:12.435+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:10:12.434+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:13.039+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:13.054+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.627 seconds
[2023-11-16T11:10:38.020+0000] {processor.py:157} INFO - Started process (PID=890) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:38.021+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:10:38.025+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:10:38.024+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:38.705+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:10:38.722+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.706 seconds
[2023-11-16T11:11:09.052+0000] {processor.py:157} INFO - Started process (PID=899) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:09.054+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:11:09.056+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:11:09.056+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:09.722+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:09.741+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.693 seconds
[2023-11-16T11:11:39.985+0000] {processor.py:157} INFO - Started process (PID=908) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:39.986+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:11:39.988+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:11:39.988+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:40.565+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:11:40.581+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.599 seconds
[2023-11-16T11:12:10.860+0000] {processor.py:157} INFO - Started process (PID=917) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:10.861+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:12:10.864+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:12:10.863+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:12.348+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:12.362+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.505 seconds
[2023-11-16T11:12:15.289+0000] {processor.py:157} INFO - Started process (PID=926) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:15.291+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:12:15.294+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:12:15.293+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:16.048+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:12:16.041+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:12:16.050+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:16.068+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.782 seconds
[2023-11-16T11:12:46.425+0000] {processor.py:157} INFO - Started process (PID=935) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:46.426+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:12:46.429+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:12:46.429+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:46.992+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:12:46.986+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:12:46.994+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:12:47.015+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.593 seconds
[2023-11-16T11:13:17.278+0000] {processor.py:157} INFO - Started process (PID=944) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:17.280+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:13:17.282+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:13:17.282+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:18.770+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:13:18.764+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:13:18.772+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:18.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.517 seconds
[2023-11-16T11:13:49.048+0000] {processor.py:157} INFO - Started process (PID=953) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:49.049+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:13:49.053+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:13:49.053+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:52.048+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:13:52.039+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:13:52.050+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:13:52.068+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 3.024 seconds
[2023-11-16T11:14:22.306+0000] {processor.py:157} INFO - Started process (PID=962) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:22.308+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:14:22.311+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:14:22.311+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:22.900+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:14:22.895+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:14:22.902+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:22.919+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.617 seconds
[2023-11-16T11:14:53.198+0000] {processor.py:157} INFO - Started process (PID=971) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:53.200+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:14:53.205+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:14:53.204+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:53.911+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:14:53.905+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:14:53.913+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:14:53.932+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.739 seconds
[2023-11-16T11:15:24.226+0000] {processor.py:157} INFO - Started process (PID=980) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:24.228+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:15:24.232+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:15:24.232+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:24.889+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:15:24.880+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:15:24.891+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:24.914+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.691 seconds
[2023-11-16T11:15:55.080+0000] {processor.py:157} INFO - Started process (PID=989) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:55.082+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:15:55.085+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:15:55.085+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:55.736+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:15:55.730+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:15:55.739+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:15:55.762+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.685 seconds
[2023-11-16T11:16:26.047+0000] {processor.py:157} INFO - Started process (PID=998) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:26.049+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:16:26.052+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:16:26.052+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:26.749+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:16:26.740+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:16:26.752+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:26.775+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.731 seconds
[2023-11-16T11:16:57.045+0000] {processor.py:157} INFO - Started process (PID=1007) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:57.047+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:16:57.050+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:16:57.050+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:57.723+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:16:57.717+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:16:57.725+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:16:57.745+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.704 seconds
[2023-11-16T11:17:27.855+0000] {processor.py:157} INFO - Started process (PID=1016) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:27.856+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:17:27.861+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:17:27.861+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:28.533+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:17:28.520+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:17:28.535+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:28.555+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.704 seconds
[2023-11-16T11:17:58.822+0000] {processor.py:157} INFO - Started process (PID=1025) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:58.825+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:17:58.829+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:17:58.829+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:59.527+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:17:59.521+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:17:59.529+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:17:59.550+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.731 seconds
[2023-11-16T11:18:29.819+0000] {processor.py:157} INFO - Started process (PID=1033) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:18:29.821+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:18:29.824+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:18:29.824+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:18:30.475+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:18:30.469+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:18:30.477+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:18:30.495+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.681 seconds
[2023-11-16T11:19:00.776+0000] {processor.py:157} INFO - Started process (PID=1042) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:00.778+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:19:00.782+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:19:00.782+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:01.463+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:19:01.457+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:19:01.466+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:01.486+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.714 seconds
[2023-11-16T11:19:31.750+0000] {processor.py:157} INFO - Started process (PID=1051) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:31.752+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:19:31.756+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:19:31.755+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:32.544+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:19:32.537+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:19:32.546+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:19:32.566+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.821 seconds
[2023-11-16T11:20:02.813+0000] {processor.py:157} INFO - Started process (PID=1061) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:02.814+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:20:02.818+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:20:02.817+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:03.368+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:20:03.361+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:20:03.370+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:03.389+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.582 seconds
[2023-11-16T11:20:33.653+0000] {processor.py:157} INFO - Started process (PID=1070) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:33.655+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:20:33.658+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:20:33.657+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:34.487+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:20:34.482+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:20:34.489+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:20:34.510+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.862 seconds
[2023-11-16T11:21:04.788+0000] {processor.py:157} INFO - Started process (PID=1079) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:04.790+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:21:04.793+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:21:04.793+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:05.522+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:21:05.516+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:21:05.524+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:05.544+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.761 seconds
[2023-11-16T11:21:35.783+0000] {processor.py:157} INFO - Started process (PID=1088) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:35.784+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:21:35.788+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:21:35.788+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:36.420+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:21:36.413+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:21:36.423+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:21:36.445+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.666 seconds
[2023-11-16T11:22:06.706+0000] {processor.py:157} INFO - Started process (PID=1097) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:06.708+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:22:06.712+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:22:06.712+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:07.319+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:22:07.312+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:22:07.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:07.342+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.639 seconds
[2023-11-16T11:22:37.604+0000] {processor.py:157} INFO - Started process (PID=1106) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:37.605+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:22:37.609+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:22:37.608+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:38.242+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:22:38.236+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:22:38.244+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:22:38.264+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.664 seconds
[2023-11-16T11:23:08.458+0000] {processor.py:157} INFO - Started process (PID=1116) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:08.462+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:23:08.466+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:23:08.466+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:09.157+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:23:09.150+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:23:09.159+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:09.189+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.734 seconds
[2023-11-16T11:23:39.708+0000] {processor.py:157} INFO - Started process (PID=1125) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:39.710+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:23:39.713+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:23:39.713+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:40.369+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:23:40.363+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:23:40.372+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:23:40.392+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.688 seconds
[2023-11-16T11:24:10.665+0000] {processor.py:157} INFO - Started process (PID=1134) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:10.666+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:24:10.670+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:24:10.669+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:11.299+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:24:11.295+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:24:11.301+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:11.319+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.659 seconds
[2023-11-16T11:24:41.587+0000] {processor.py:157} INFO - Started process (PID=1143) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:41.589+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:24:41.594+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:24:41.593+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:42.224+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:24:42.218+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:24:42.226+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:24:42.244+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.661 seconds
[2023-11-16T11:25:12.470+0000] {processor.py:157} INFO - Started process (PID=1152) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:12.471+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:25:12.473+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:25:12.473+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:13.015+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:25:13.009+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:25:13.018+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:13.042+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.576 seconds
[2023-11-16T11:25:43.302+0000] {processor.py:157} INFO - Started process (PID=1161) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:43.303+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:25:43.306+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:25:43.306+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:43.847+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:25:43.842+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:25:43.849+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:25:43.868+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.570 seconds
[2023-11-16T11:26:13.980+0000] {processor.py:157} INFO - Started process (PID=1170) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:13.982+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:26:13.985+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:26:13.985+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:14.624+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:26:14.618+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:26:14.626+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:14.649+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.673 seconds
[2023-11-16T11:26:44.925+0000] {processor.py:157} INFO - Started process (PID=1179) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:44.926+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:26:44.929+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:26:44.929+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:45.557+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:26:45.550+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:26:45.559+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:26:45.587+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.666 seconds
[2023-11-16T11:27:15.871+0000] {processor.py:157} INFO - Started process (PID=1189) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:15.880+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:27:15.890+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:27:15.889+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:16.528+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:27:16.522+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:27:16.530+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:16.557+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.691 seconds
[2023-11-16T11:27:46.803+0000] {processor.py:157} INFO - Started process (PID=1203) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:46.805+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:27:46.808+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:27:46.807+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:47.496+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:27:47.489+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:27:47.499+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:27:47.522+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.722 seconds
[2023-11-16T11:28:17.792+0000] {processor.py:157} INFO - Started process (PID=1212) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:17.793+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:28:17.796+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:28:17.796+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:18.475+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:28:18.469+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:28:18.477+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:18.503+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.715 seconds
[2023-11-16T11:28:48.748+0000] {processor.py:157} INFO - Started process (PID=1221) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:48.749+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T11:28:48.751+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:28:48.751+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:49.210+0000] {logging_mixin.py:151} INFO - [2023-11-16T11:28:49.205+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T11:28:49.211+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T11:28:49.225+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.479 seconds
[2023-11-16T12:33:11.178+0000] {processor.py:157} INFO - Started process (PID=1223) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:11.182+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:33:11.193+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:33:11.190+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:14.022+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:33:14.012+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:33:14.028+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:14.055+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 2.880 seconds
[2023-11-16T12:33:44.277+0000] {processor.py:157} INFO - Started process (PID=1232) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:44.279+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:33:44.281+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:33:44.281+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:44.833+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:33:44.826+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:33:44.835+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:33:44.854+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.580 seconds
[2023-11-16T12:34:15.103+0000] {processor.py:157} INFO - Started process (PID=1241) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:15.104+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:34:15.106+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:34:15.106+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:15.575+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:34:15.569+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:34:15.577+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:15.591+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.492 seconds
[2023-11-16T12:34:45.699+0000] {processor.py:157} INFO - Started process (PID=1250) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:45.701+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:34:45.705+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:34:45.704+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:46.432+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:34:46.425+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:34:46.435+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:34:46.466+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.771 seconds
[2023-11-16T12:35:16.723+0000] {processor.py:157} INFO - Started process (PID=1259) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:16.724+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:35:16.727+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:35:16.726+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:17.263+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:35:17.255+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:35:17.265+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:17.293+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.574 seconds
[2023-11-16T12:35:47.547+0000] {processor.py:157} INFO - Started process (PID=1268) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:47.548+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:35:47.552+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:35:47.551+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:48.232+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:35:48.225+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:35:48.234+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:35:48.258+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.716 seconds
[2023-11-16T12:36:18.466+0000] {processor.py:157} INFO - Started process (PID=1277) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:18.468+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:36:18.471+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:36:18.471+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:19.095+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:36:19.089+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:36:19.097+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:19.116+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.653 seconds
[2023-11-16T12:36:49.378+0000] {processor.py:157} INFO - Started process (PID=1286) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:49.380+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:36:49.384+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:36:49.383+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:50.067+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:36:50.060+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:36:50.070+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:36:50.091+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.718 seconds
[2023-11-16T12:37:20.353+0000] {processor.py:157} INFO - Started process (PID=1295) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:20.354+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:37:20.357+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:37:20.357+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:21.044+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:37:21.038+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:37:21.046+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:21.130+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.782 seconds
[2023-11-16T12:37:51.199+0000] {processor.py:157} INFO - Started process (PID=1304) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:51.200+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:37:51.203+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:37:51.202+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:51.848+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:37:51.843+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:37:51.850+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:37:51.902+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.707 seconds
[2023-11-16T12:38:22.187+0000] {processor.py:157} INFO - Started process (PID=1313) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:22.189+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:38:22.191+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:38:22.191+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:22.815+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:38:22.809+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:38:22.817+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:22.873+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.688 seconds
[2023-11-16T12:38:53.128+0000] {processor.py:157} INFO - Started process (PID=1322) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:53.130+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:38:53.132+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:38:53.132+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:53.757+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:38:53.752+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:38:53.759+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:38:53.778+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.654 seconds
[2023-11-16T12:39:24.042+0000] {processor.py:157} INFO - Started process (PID=1331) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:24.043+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:39:24.046+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:39:24.045+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:24.579+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:39:24.572+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:39:24.581+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:24.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.565 seconds
[2023-11-16T12:39:54.859+0000] {processor.py:157} INFO - Started process (PID=1340) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:54.861+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:39:54.864+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:39:54.864+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:55.532+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:39:55.526+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:39:55.535+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:39:55.632+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.777 seconds
[2023-11-16T12:40:25.816+0000] {processor.py:157} INFO - Started process (PID=1349) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:25.818+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:40:25.821+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:40:25.821+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:26.485+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:40:26.479+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:40:26.487+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:26.521+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.709 seconds
[2023-11-16T12:40:56.792+0000] {processor.py:157} INFO - Started process (PID=1358) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:56.794+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:40:56.797+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:40:56.797+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:57.443+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:40:57.436+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:40:57.445+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:40:57.468+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.679 seconds
[2023-11-16T12:41:27.728+0000] {processor.py:157} INFO - Started process (PID=1367) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:27.729+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:41:27.733+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:41:27.732+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:28.396+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:41:28.388+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:41:28.398+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:28.469+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.745 seconds
[2023-11-16T12:41:58.554+0000] {processor.py:157} INFO - Started process (PID=1376) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:58.556+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:41:58.560+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:41:58.560+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:59.282+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:41:59.275+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:41:59.284+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:41:59.304+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.754 seconds
[2023-11-16T12:42:29.575+0000] {processor.py:157} INFO - Started process (PID=1385) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:42:29.577+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:42:29.580+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:42:29.579+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:42:30.660+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:42:30.653+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:42:30.662+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:42:30.686+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.116 seconds
[2023-11-16T12:43:00.811+0000] {processor.py:157} INFO - Started process (PID=1401) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:00.812+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:43:00.814+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:43:00.814+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:01.402+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:43:01.396+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:43:01.404+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:01.425+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.618 seconds
[2023-11-16T12:43:31.725+0000] {processor.py:157} INFO - Started process (PID=1410) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:31.726+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:43:31.728+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:43:31.728+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:32.326+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:43:32.321+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:43:32.327+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:43:32.390+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.668 seconds
[2023-11-16T12:44:02.636+0000] {processor.py:157} INFO - Started process (PID=1419) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:02.639+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:44:02.643+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:44:02.642+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:03.306+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:44:03.300+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:44:03.308+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:03.330+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.699 seconds
[2023-11-16T12:44:33.614+0000] {processor.py:157} INFO - Started process (PID=1428) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:33.615+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:44:33.618+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:44:33.618+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:34.317+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:44:34.310+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:44:34.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:44:34.355+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.745 seconds
[2023-11-16T12:45:04.619+0000] {processor.py:157} INFO - Started process (PID=1437) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:04.621+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:45:04.625+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:45:04.625+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:05.307+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:45:05.301+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:45:05.309+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:05.341+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.726 seconds
[2023-11-16T12:45:35.637+0000] {processor.py:157} INFO - Started process (PID=1446) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:35.638+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:45:35.642+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:45:35.641+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:36.221+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:45:36.214+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:45:36.223+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:45:36.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.611 seconds
[2023-11-16T12:46:06.313+0000] {processor.py:157} INFO - Started process (PID=1455) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:06.314+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:46:06.317+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:46:06.317+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:07.379+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:46:07.373+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:46:07.381+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:07.402+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.093 seconds
[2023-11-16T12:46:37.559+0000] {processor.py:157} INFO - Started process (PID=1464) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:37.561+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:46:37.599+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:46:37.598+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:38.526+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:46:38.521+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:46:38.527+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:46:38.544+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.990 seconds
[2023-11-16T12:47:08.822+0000] {processor.py:157} INFO - Started process (PID=1473) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:08.824+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:47:08.828+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:47:08.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:09.907+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:47:09.900+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:47:09.909+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:09.933+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.114 seconds
[2023-11-16T12:47:40.071+0000] {processor.py:157} INFO - Started process (PID=1482) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:40.073+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:47:40.076+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:47:40.076+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:40.665+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:47:40.658+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:47:40.668+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:47:40.701+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.634 seconds
[2023-11-16T12:48:10.973+0000] {processor.py:157} INFO - Started process (PID=1491) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:10.975+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:48:10.980+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:48:10.978+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:11.595+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:48:11.589+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:48:11.597+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:11.676+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.708 seconds
[2023-11-16T12:48:41.968+0000] {processor.py:157} INFO - Started process (PID=1500) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:41.970+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:48:41.973+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:48:41.972+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:42.611+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:48:42.605+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:48:42.613+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:48:42.638+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.674 seconds
[2023-11-16T12:49:12.828+0000] {processor.py:157} INFO - Started process (PID=1509) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:12.829+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:49:12.831+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:49:12.831+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:13.442+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:49:13.434+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:49:13.445+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:13.469+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.645 seconds
[2023-11-16T12:49:43.781+0000] {processor.py:157} INFO - Started process (PID=1518) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:43.783+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:49:43.787+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:49:43.786+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:44.938+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:49:44.933+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:49:44.940+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:49:44.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.196 seconds
[2023-11-16T12:50:15.289+0000] {processor.py:157} INFO - Started process (PID=1527) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:15.290+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:50:15.293+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:50:15.292+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:15.916+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:50:15.912+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:50:15.918+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:15.936+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.652 seconds
[2023-11-16T12:50:46.199+0000] {processor.py:157} INFO - Started process (PID=1536) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:46.201+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:50:46.205+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:50:46.204+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:46.879+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:50:46.870+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:50:46.882+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:50:46.906+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.710 seconds
[2023-11-16T12:51:17.212+0000] {processor.py:157} INFO - Started process (PID=1545) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:17.214+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:51:17.218+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:51:17.217+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:18.028+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:51:18.021+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:51:18.030+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:18.048+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.841 seconds
[2023-11-16T12:51:48.281+0000] {processor.py:157} INFO - Started process (PID=1554) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:48.283+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:51:48.286+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:51:48.286+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:49.582+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:51:49.574+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:51:49.584+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:51:49.604+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 1.327 seconds
[2023-11-16T12:52:19.847+0000] {processor.py:157} INFO - Started process (PID=1563) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:19.849+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:52:19.851+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:52:19.851+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:20.363+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:52:20.357+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:52:20.366+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:20.387+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.543 seconds
[2023-11-16T12:52:50.650+0000] {processor.py:157} INFO - Started process (PID=1572) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:50.652+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:52:50.654+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:52:50.654+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:51.289+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:52:51.283+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:52:51.292+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:52:51.318+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.671 seconds
[2023-11-16T12:53:21.567+0000] {processor.py:157} INFO - Started process (PID=1581) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:21.569+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:53:21.572+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:53:21.571+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:22.163+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:53:22.155+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:53:22.165+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:22.187+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.624 seconds
[2023-11-16T12:53:52.460+0000] {processor.py:157} INFO - Started process (PID=1590) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:52.462+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:53:52.464+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:53:52.464+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:53.261+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:53:53.254+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 50, in <module>
    extract >> transform >> load
NameError: name 'transform' is not defined
[2023-11-16T12:53:53.263+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:53:53.287+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.830 seconds
[2023-11-16T12:54:23.599+0000] {processor.py:157} INFO - Started process (PID=1599) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:23.600+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:54:23.603+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:54:23.603+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:23.630+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:54:23.629+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:54:23.631+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:23.654+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.060 seconds
[2023-11-16T12:54:53.840+0000] {processor.py:157} INFO - Started process (PID=1607) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:53.841+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:54:53.844+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:54:53.844+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:53.869+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:54:53.868+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:54:53.870+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:54:53.892+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.055 seconds
[2023-11-16T12:55:24.173+0000] {processor.py:157} INFO - Started process (PID=1615) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:24.174+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:55:24.178+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:55:24.177+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:24.209+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:55:24.205+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:55:24.211+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:24.261+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.092 seconds
[2023-11-16T12:55:54.410+0000] {processor.py:157} INFO - Started process (PID=1623) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:54.412+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:55:54.415+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:55:54.414+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:54.445+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:55:54.443+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:55:54.446+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:55:54.473+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.066 seconds
[2023-11-16T12:56:24.658+0000] {processor.py:157} INFO - Started process (PID=1631) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:24.659+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:56:24.661+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:56:24.660+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:24.676+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:56:24.675+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:56:24.676+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:24.691+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.036 seconds
[2023-11-16T12:56:54.842+0000] {processor.py:157} INFO - Started process (PID=1639) to work on /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:54.842+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dag_etl.py for tasks to queue
[2023-11-16T12:56:54.844+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:56:54.844+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:54.859+0000] {logging_mixin.py:151} INFO - [2023-11-16T12:56:54.858+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dag_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl.py", line 11
    def extract_data() {
                       ^
SyntaxError: invalid syntax
[2023-11-16T12:56:54.860+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl.py
[2023-11-16T12:56:54.872+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dag_etl.py took 0.032 seconds
